.travis.yml

```diff
@@ -10,8 +10,8 @@ before_install:
   - travis_retry npm install --loglevel=http
   - >
     openssl aes-256-cbc \
-      -K $encrypted_4fb8e306b739_key \
-      -iv $encrypted_4fb8e306b739_iv \
```

Comment:
@indexzero it seems like best practice for unit tests is create mock interfaces for external service dependencies. Is there any reason you coded these to require a call the actual Loggly service? If not, I agree with @psquickitjayant. The test cases will run faster and be a better test of code correctness. Are you ok with this if we converted the test cases over to mocks and submitted an updated PR?